app/policies/transaction_policy.rb

```diff
@@ -27,6 +30,12 @@ def destroy?
     record.activity_id.present?
   end
 
+  private def activity_is_project_level?
```

Comment:
Wondering if we can replace this method by using `record.project?` in the public methods instead?

Reply:
`record` at this point is a transaction, not an activity. For Transaction create/update we need to get the Activity and then see if it's a project